src/query/service/src/servers/flight/flight_service.rs

```diff
@@ -1,3 +1,4 @@
+// Copyright 2016-2019 The Apache Software Foundation
```

Comment:
How can we more friendly reference this code? cc: @Xuanwo

Reply:
The current way looks good enough.